src/app/types/record-funding.endpoint.ts

```diff
@@ -44,7 +44,7 @@ export interface Funding extends AssertionBase {
   region: Value
   country: Value
   countryForDisplay?: string
-  fundingTypeForDisplay?: string
+  TypeForDisplay?: string
```

Comment:
there is a typo here 
```suggestion
  typeForDisplay?: string
```